roxie/ccd/ccdmain.cpp

```diff
@@ -670,6 +671,12 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         useOldTopology = checkFileExists(topologyFile.str());
         topology = loadConfiguration(useOldTopology ? nullptr : defaultYaml, argv, "roxie", "ROXIE", topologyFile, nullptr, "@netAddress");
         saveTopology();
+        if (topology->getPropBool("expert/@profileStartup", false))
+        {
+            double interval = topology->getPropReal("expert/@profileStartupInterval", 0.2);
```

Comment:
This should work the same in bare-metal, correct ?

Reply:
I don't see why not, though I don't know if bare-metal configmgr would make it easy to set this (or any other "expert" flags in Roxie or Thor).